concourse-chrome-driver/terraform/deployments/670214072732/module-codebuild-get-changed-file-list.tf

```diff
@@ -0,0 +1,16 @@
+module "codebuild-get-changed-file-list" {
+  source                      = "github.com/alphagov/cyber-security-shared-terraform-modules//codebuild/codebuild_get_changed_file_list"
+  codebuild_service_role_name = var.codebuild_service_role_name
+  deployment_account_id       = data.aws_caller_identity.current.account_id
+  deployment_role_name        = "CodePipelineDeployerRole_${data.aws_caller_identity.current.account_id}"
+  codebuild_image             = var.codebuild_image
+  pipeline_name               = var.pipeline_name
+  stage_name                  = "Changes"
+  action_name                 = "GetChangedFiles"
+  environment                 = var.environment
+  github_pat                  = var.github_pat
+  repo_name                   = var.repo_name
+  docker_hub_credentials      = var.docker_hub_credentials
+  output_artifact_path        = var.output_artifact_path
```

Comment:
This is the problem. What that module was doing before was writing the files into a custom path in the artifact bucket. What it does now is write to a named file in the designated output artifact. 

So `output_artifact_path` for `get_changed_files` should be set to `changed_files.json` or omitted. 
and `output_artifact_path` for `get_actions_required` should be set to `actions_required.json` or omitted.

The defaults should just work if you don't set the vars.